modules/Behaviour/moduleFunctions.php

```diff
@@ -56,7 +56,7 @@ function getBehaviourRecord(ContainerInterface $container, $gibbonPersonID)
             $table = DataTable::createPaginated('behaviour'.$schoolYear['gibbonSchoolYearID'], $criteria);
             $table->setTitle($schoolYear['name']);
 
-            if ($schoolYear['gibbonSchoolYearID'] == $_SESSION[$guid]['gibbonSchoolYearID']) {
+            if ($schoolYear['gibbonSchoolYearID'] == $session->get('gibbonSchoolYearID')) {
```

Comment:
We can't make changes to the moduleFunctions.php script at this time. Can you revert these changes please? Thanks!